plugins/out_opensearch/opensearch.c

```diff
@@ -1063,6 +1063,11 @@ static struct flb_config_map config_map[] = {
      0, FLB_FALSE, 0,
      "External ID for the AWS IAM Role specified with `aws_role_arn`"
     },
+    {
+     FLB_CONFIG_MAP_STR, "aws_service_name", "es",
```

Comment:
the service name for aws opensearch serverless is "aoss". "es" doesn't work.

Reply:
This plugin is meant to be used with serverless and non-serverless. Non-serverless (es) has always been the default. This change allows you to configure something different.

Reply:
Please see our example here @seaHour https://github.com/aws-samples/amazon-ecs-firelens-examples/tree/mainline/examples/fluent-bit/amazon-opensearch-serverless